book/getting-started/run-a-wallaroo-application-docker.md

```diff
@@ -19,13 +19,25 @@ Let's get started!
 
 ## Terminal 1, Start the Wallaroo Docker container
 
-```bash
+{% codetabs name="UNIX Bash", type="bash" -%}
 docker run --rm -it --privileged -p 4000:4000 \
 -v /tmp/wallaroo-docker/wallaroo-src:/src/wallaroo \
 -v /tmp/wallaroo-docker/python-virtualenv:/src/python-virtualenv \
 --name wally \
 wallaroo-labs-docker-wallaroolabs.bintray.io/{{ docker_version_url }}
-```
+{%- language name="Windows Powershell", type="bash" -%}
+docker run --rm -it --privileged -p 4000:4000 |
+-v /tmp/wallaroo-docker/wallaroo-src:/src/wallaroo |
+-v /tmp/wallaroo-docker/python-virtualenv:/src/python-virtualenv |
```

Comment:
These volume specifiers don't work. You'll want to map them to an existing directory. It might be best to have the user create a workspace like:
```
mkdir wallaroo-application
cd wallaroo-application
mkdir python-virtualenv
```
and then use:
```
...
-v %CD%:/src/wallaroo
-v %CD%\python-virtualenv:/src/python-virtualenv
... 
```